tests/syscall/fs/enc/enc.cpp

```diff
@@ -637,17 +637,23 @@ void test_fs(const char* src_dir, const char* tmp_dir)
         OE_TEST(
             oe_mount("/", "/", OE_DEVICE_NAME_HOST_FILE_SYSTEM, 0, NULL) == 0);
         const int flags = OE_O_CREAT | OE_O_TRUNC | OE_O_WRONLY;
-        OE_TEST(oe_open(path, flags, MODE) == 0);
+        OE_TEST(oe_open(path, flags, MODE) != -1);
```

Comment:
Good change on checking return value of oe_open, I didn't even notice this the first time around!

Reply:
And good change to all the other expected return values too!
Hmm. We might want to add function headers to each of those functions that specify expected return types. (Not in this PR) I guess they're supposed to be equivalent to libc function calls, though.